pkgs/tools/misc/flitter/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, ocamlPackages
+, fetchFromGitHub
+, makeWrapper
+, python3
+}:
+
+ocamlPackages.buildDunePackage rec {
```

Comment:
One question, why not also use `buildDunePackage` here like the other module?

Reply:
in ocaml-packages.nix callPackage is different: it can pass ocaml dependencies and buildDunePackage. all-packages.nix's callPackage only passes top-level packages, so it's either this or an extra argument to callPackage, but I think this is better

Reply:
Or I could also use ocamlPackages.callPackage in all-packages.nix, but it'd be the only occurrence of such a pattern and it'd make overriding the ocaml packages more difficult

Reply:
Is there a reason to add `flitter` to `all-packages` instead of `ocaml-modules`? Is it because it's not a library? I see some packages (`hol_light` is an example) that are defined in `ocaml-packages` and then inherited in `all-packages`, but I'm not sure if there is a policy for this or if one of the patterns is more common.